snuba/query/processors/type_converters/fixedstring_array_column_processor.py

```diff
@@ -0,0 +1,39 @@
+from snuba.query.expressions import (
+    Column,
+    Expression,
+    FunctionCall,
+    Literal,
+    Lambda,
+    Argument,
+)
+from snuba.query.processors.type_converters import BaseTypeConverter, ColumnTypeError
+
+
+class FixedStringArrayColumnProcessor(BaseTypeConverter):
+    def _translate_literal(self, exp: Literal) -> Expression:
+        try:
+            assert isinstance(exp.value, str)
+            return FunctionCall(
+                exp.alias,
+                "toFixedString",
+                (Literal(None, value=exp.value), Literal(None, 32)),
+            )
+        except (AssertionError, ValueError):
+            raise ColumnTypeError("Not a valid UUID string")
+
+    def _process_expressions(self, exp: Expression) -> Expression:
+        if isinstance(exp, Column) and exp.column_name in self.columns:
+            return FunctionCall(
```

Comment:
Do we even need this? Does Sentry care if it gets a FixedString or String back?

Reply:
I guess something like `has(hierarchical_hashes, <thing that is not a literal>)` would need this but I can remove this since it's just unnecessary cost in query results for something that will never happen.